Fortinet/fortigate/_meta/smart-descriptions.json

```
@@ -456,4 +456,462 @@
             }
         ]
     }
-]
\ No newline at end of file
+][
```

Thank you @Men-hau for this work !

```suggestion
][
```

there might be an issue here

please use the custom intake view to test your changes and update this PR accordingly